MediaBrowser.Model/Updates/PackageInfo.cs

```diff
@@ -87,90 +39,24 @@ public class PackageInfo
         public string category { get; set; }
 
         /// <summary>
-        /// Gets or sets the catalog tile color.
-        /// </summary>
-        /// <value>The owner.</value>
-        public string tileColor { get; set; }
-
-        /// <summary>
-        /// Gets or sets the feature id of this package (if premium).
-        /// </summary>
-        /// <value>The feature id.</value>
-        public string featureId { get; set; }
-
-        /// <summary>
-        /// Gets or sets the registration info for this package (if premium).
-        /// </summary>
-        /// <value>The registration info.</value>
-        public string regInfo { get; set; }
-
-        /// <summary>
-        /// Gets or sets the price for this package (if premium).
-        /// </summary>
-        /// <value>The price.</value>
-        public float price { get; set; }
-
-        /// <summary>
-        /// Gets or sets the target system for this plug-in (Server, MBTheater, MBClassic).
-        /// </summary>
-        /// <value>The target system.</value>
-        public PackageTargetSystem targetSystem { get; set; }
-
-        /// <summary>
-        /// The guid of the assembly associated with this package (if a plug-in).
+        /// The guid of the assembly associated with this plugin.
         /// This is used to identify the proper item for automatic updates.
         /// </summary>
         /// <value>The name.</value>
         public string guid { get; set; }
```

Comment:
I think this should really be `Guid guid {get;set;}`

Reply:
same as above